Move deprecated comment to DESCRIPTION, since having it the NAME
section confuses some apropos implementations.

diff -Naur perl-5.6.1.orig/lib/I18N/Collate.pm perl-5.6.1/lib/I18N/Collate.pm
--- perl-5.6.1.orig/lib/I18N/Collate.pm	Fri Feb 23 13:57:55 2001
+++ perl-5.6.1/lib/I18N/Collate.pm	Fri Jan 11 01:10:08 2002
@@ -4,6 +4,15 @@
 
 I18N::Collate - compare 8-bit scalar data according to the current locale
 
+=head1 SYNOPSIS
+
+    use I18N::Collate;
+    setlocale(LC_COLLATE, 'locale-of-your-choice'); 
+    $s1 = new I18N::Collate "scalar_data_1";
+    $s2 = new I18N::Collate "scalar_data_2";
+
+=head1 DESCRIPTION
+
   ***
 
   WARNING: starting from the Perl version 5.003_06
@@ -20,15 +29,6 @@
   See the perllocale manual page for further information.
 
   ***
-
-=head1 SYNOPSIS
-
-    use I18N::Collate;
-    setlocale(LC_COLLATE, 'locale-of-your-choice'); 
-    $s1 = new I18N::Collate "scalar_data_1";
-    $s2 = new I18N::Collate "scalar_data_2";
-
-=head1 DESCRIPTION
 
 This module provides you with objects that will collate 
 according to your national character set, provided that the 
